wiki/Tournaments/oSEAT/2/en.md

```diff
@@ -97,38 +97,31 @@ The osu! South East Tournament 2 is run by various community members.
 
 -----------------------
 
-## Match Schedules: Finals
+## Match Schedules: Grand Finals
 
-### Friday, 18 January 2019
+### Saturday, 26 January 2019
 
 | Match Code | Player A |  |  | Player B | Match Time (UTC+8) |
 | :-: | --: | :-: | :-: | :-- | :-: |
-| MB | Crezz | ![][flag_ID] | ![][flag_SG] | GSBlank | **22:00** |
+| LB | Rexeez | ![][flag_ID] | ![][flag_MY] | GSBlank | **21:00** |
 
-### Saturday, 19 January 2019
-
-| Match Code | Player A |  |  | Player B | Match Time (UTC+8) |
-| :-: | --: | :-: | :-: | :-- | :-: |
-| MA | Raindrop | ![][flag_SG] | ![][flag_SG] | SeeL | **22:00** |
-
-### Sunday, 20 January 2019
+### Sunday, 27 January 2019
 
 | Match Code | Player A |  |  | Player B | Match Time (UTC+8) |  |
 | :-: | --: | :-: | :-: | :-- | :-: | :-: |
-| LA | Rexeez | ![][flag_ID] | ![][flag_ID] | Skydiver | **13:30** |  |
-| NA-1 | Raindrop | ![][flag_SG] | ![][flag_MY] | Crezz | **13:30** | ¹ |
-| NA-2 | Raindrop | ![][flag_SG] | ![][flag_MY] | GSBlank | **22:00** | ¹ |
-| NA-3 | SeeL | ![][flag_SG] | ![][flag_MY] | Crezz | **13:30** | ¹ |
-| NA-4 | SeeL | ![][flag_SG] | ![][flag_MY] | GSBlank | **22:00** | ¹ |
+| LC-1 | Skydiver | ![][flag_ID] | ![][flag_ID] | Rexeez | **21:00** | ¹ |
+| LC-2 | Skydiver | ![][flag_ID] | ![][flag_MY] | GSBlank | **21:00** | ¹ |
 
-¹ Encounter dependent on winner of Match MA and Match MB
+¹ Encounter dependent on winner of Match LB
 
 -----------------------
 
 ## Mappools
 
 ### Finals
 
+ **This mappool will be played in Finals and Grand Finals.**
```

Comment:
I don't think you should bold here since it seems reserved for the download links

just a style thing, either way works probably